zerver/lib/email_mirror.py

```diff
@@ -155,6 +155,8 @@ def create_missed_message_address(user_profile: UserProfile, message: Message) -
 def construct_zulip_body(
     message: EmailMessage,
     realm: Realm,
+    sender: UserProfile,
+    *,
```

Comment:
Should we make `sender` a kwarg already? I feel it's already hard to track which arg means what when they're `(message, realm, user_profile)`

Reply:
Sure; repushed.